docs/reference/ml/trained-models/apis/start-trained-model-deployment.asciidoc

```diff
@@ -55,6 +55,11 @@ include::{es-repo-dir}/ml/ml-shared.asciidoc[tag=model-id]
 [[start-trained-model-deployment-query-params]]
 == {api-query-parms-title}
 
+`cache_size`::
+(Optional, <<byte-units,byte value>>)
+The inference cache size (off JVM heap) per node for the model. The default value is the same size as the model
+size estimate. To disable cache, `0b` can be provided.
```

Comment:
> model size estimate. 

Is that the same as the `model_size_bytes` returned by the get trained models API? If yes, it might be helpful to reference that here.